sale_advance_payment/wizard/sale_advance_payment_wzd.py

```diff
@@ -99,24 +103,26 @@ def default_get(self, fields_list):
                 {
                     "order_id": sale.id,
                     "amount_total": sale.amount_residual,
-                    "currency_id": sale.pricelist_id.currency_id.id,
+                    "currency_id": sale.pricelist_id.currency_id.id
+                    or sale.currency_id.id,
```

Comment:
https://github.com/odoo/odoo/blob/17.0/addons/sale/models/sale_order.py#L392-L401
The pricelist is not set anymore on draft SO. 
The Travis issue might come from this. 
I've ran the tests on the module in its version 16.0, and the currency_id doesn't match with the one of 17.0